src/openforms/submissions/templates/admin/submissions/submission/logs.html

```diff
@@ -1,8 +1,8 @@
 {% extends "admin/change_form.html" %}
-{% load i18n admin_urls static %}
+{% load i18n static admin_urls admin_list forms_admin_list %}
```

Comment:
`forms_admin_list` is unused, nor is `admin_list` or `admin_urls` I tink

```suggestion
{% load i18n static %}
```

Reply:
admin_urls is used with the admin_urlname filter